backend/census_historical_migration/sac_general_lib/report_id_generator.py

```diff
@@ -1,8 +1,13 @@
-from census_historical_migration.sac_general_lib.utils import _census_date_to_datetime
+from census_historical_migration.sac_general_lib.utils import (
+    _census_date_to_datetime,
+    xform_census_date_to_datetime,
+)
 
 
 # FIXME: Get the padding/shape right on the report_id
-def dbkey_to_report_id(Gen, dbkey):
+def dbkey_to_report_id(
+    Gen, dbkey
+):  # FIXME-MSHD: Remove this function once we switch all workbook generators to using census models
     g = Gen.select(Gen.audityear, Gen.fyenddate).where(Gen.dbkey == dbkey).get()

```

Comment:
Let's switch to django ORM

Reply:
This function is used by other parts of the code that are not related to this PR. There is another ticket to remove it and clean the code (I added comment on top)